spec/features/get_ready_for_brexit_checker_request_spec.rb

```diff
@@ -11,7 +11,21 @@
   end
 
   scenario "sucessful request for changing a result" do
+    request = expect_zendesk_to_receive_ticket(
+      "subject" => "Get ready for Brexit checker",
+      "requester" => hash_including("name" => "John Smith", "email" => "john.smith@agency.gov.uk"),
+      "comment" => {
+        "body" =>
+"[Change title]
+Action title
+
+[Change description]
+Action description"
+      }
+    )
```

Comment:
I think this needs indentation.

Reply:
I had originally thought so too, but it seems if you indent then it affects the API request that's made, so you get extra whitespace, like this:
`"comment"=>{"body"=>"[Action to change]\n          Action title\n\n          [Description of change]\n          Action description"`

When it should be this:
`"comment":{"body":"[Action to change]\nAction title\n\n[Description of change]\nAction description"`

There is precedent for this in the app, for example: https://github.com/alphagov/support/blob/master/spec/features/eu_exit_business_readiness_request_spec.rb#L21